### ISSUE TITLE
Issue with foreign key as part of the primary child key.

### ISSUE THREAD

**Issue by bertBruynooghe** [vuex-orm/vuex-orm#320]
Multiple insertion with same parent goes wrong if foreign key is part of the primary key:

```
const Vue = require('vue')
const Vuex = require('vuex')
const VuexORM = require('@vuex-orm/core')

Vue.use(Vuex)

class Post extends VuexORM.Model {
  static fields () {
    return {
      id: this.attr(),
      comments: this.hasMany('comments', 'postId')
    }
  }
}

Post.entity = 'posts'

class Comment extends VuexORM.Model {
  static fields () {
    return {
      id: this.attr(),
      postId: this.attr()
    }
  }
}

Comment.entity = 'comments'
Comment.primaryKey = ['id', 'postId']

describe('VuexORM', () => {
  beforeEach(() => {
    const database = new VuexORM.Database()

    database.register(Post, {})
    database.register(Comment, {})

    new Vuex.Store({ // eslint-disable-line no-new
      plugins: [VuexORM.install(database)],
      strict: true
    })
  })

  it('supports nested comments with composite keys', async () => {
    const data = [
      { id: 1, post: { id: 1 } },
      { id: 1, post: { id: 2 } }
    ]
    const { comments } = await Comment.create({ data })
    expect(comments.length).toEqual(2) // fails!!!
  })

  it('supports nested comments with composite keys reversed', async () => {
    const data = [
      { id: 1, comments: [{ id: 1, postId: 1 }] },
      { id: 2, comments: [{ id: 1, postId: 2 }] }
    ]
    const { comments } = await Post.create({ data })
    expect(comments.length).toEqual(2) // fails!!!
  })
})
```

**Comment by kkyouhei** [comment]
Thank you for report!
I’m going to look your report in a few day.

**Comment by kkyouhei** [comment]
I found first aid.
add post_id to Comment data.

```
    const data = [
      { id: 1, post_id: 1, post: { id: 1 } },
      { id: 1, post_id: 2, post: { id: 2 } }
    ]
```

**Comment by bertBruynooghe** [comment]
Yes, the second testcase seems to be fixed by 
https://github.com/vuex-orm/vuex-orm/pull/306

As for the first aid: it is exactly to avoid this kind of things that I choose to use Vuex ORM ;) 
But it could come as a stopgap solution for now ...

**Comment by kkyouhei** [comment]
Sorry, But if you need a quick fix please use first aid.
I'll find out how to fix in the meantime.

**Comment by kkyouhei** [comment]
my report

Record will be one in ` Normalizer.process`, because I'm thinking about modify ProcessStrategy.getId .
But i don't have good idea...

**Comment by kiaking** [comment]
This is now fixed with Vuex ORM 0.31.10 thanks to @kkyouhei! 🔥 
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.31.10
